Sources/Prometheus/MetricTypes/Summary.swift

```diff
@@ -72,9 +72,13 @@ public class Summary<NumType: DoubleRepresentable, Labels: SummaryLabels>: Metri
     
     /// Gets the metric string for this Summary
     ///
+    /// - Parameters:
+    ///     - done: Completion handler
+    ///     - metric: String value in prom-format
+    ///
     /// - Returns:
     ///     Newline seperated Prometheus formatted metric string
-    public func getMetric(_ done: @escaping (String) -> Void) {
+    public func getMetric(_ done: @escaping (_ metric: String) -> Void) {
```

Comment:
I prefer not to name closure parameters.
```suggestion
    public func getMetric(_ done: @escaping (String) -> Void) {
```